style.css

```diff
@@ -0,0 +1,356 @@
+  background: #ebebff;
+  margin-right: 8px;
+  border-radius: 8px;
+  list-style: none;
+}
+
+.card-title li:first-child {
+  list-style: none;
+  color: var(--color-three);
+}
+
+button {
+  font-weight: 500;
+  font-size: 17px;
+  margin: 28px 0;
+  width: 124px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fff;
+  color: var(--color-four);
+  border-radius: 8px;
+  border: 1px solid var(--color-one);
+}
+
+.alter {
+  color: #fff;
+  background-color: var(--bg-two);
+}
+
+/* about */
+
+.about-content-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto;
+  column-gap: 48px;
+  padding: 142px;
+  background-color: var(--default);
+  border-top-right-radius: 100px;
+}
+
+.about-title {
+  color: var(--color-three);
+  font-size: 40px;
+  font-weight: bold;
+  margin: 12px 0;
+}
+
+.lang-skills {
+  padding: 24px 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.skill-img {
+  width: 122px;
+  height: 120px;
+  margin-right: 12px;
+}
+
+.icon-colors {
+  color: var(--color-three);
+}
+
+.foot-icons {
+  padding-left: 10px;
+}
+
+hr {
+  height: 1px;
+  background-color: var(--vector);
+  opacity: 0.2;
+}
+
+.see-resume {
+  width: 162px;
+}
+
+/* footer */
+
+footer {
+  background-color: var(--default);
+  margin-top: -103px;
+}
+
+.container {
+  background-image: url(images/contact-background.png);
+  background-size: auto;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  display: flex;
+  padding: 105px 24px 8px 24px;
+  margin-top: 103px;
+  border-top-left-radius: 100px;
+}
+
+.container section {
+  margin: 0 auto;
+}
+
+.container h1 {
+  font-weight: bold;
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 12px;
+}
+
+.container p {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  margin-bottom: 40px;
+}
+
+.container input {
+  border: 1px solid #cfd8dc;
+  margin-bottom: 20px;
+  width: calc(100% - 16px);
+  padding-left: 16px;
+  height: 48px;
+  font-size: 17px;
+  color: #172b4d;
+}
+
+.container input::placeholder {
+  color: #172b4d;
+}
+
+.contacts-form {
+  width: 447px;
+  margin: 0 auto 127px auto;
+}
+
+.contacts-form textarea {
+  width: calc(100% - 16px);
+  padding: 15px 38px 15px 16px;
+  color: #172b4d;
+  font-size: 15px;
+}
+
+.contacts-form textarea::placeholder {
+  color: #b3bac5;
+}
+
+.get-it-touch {
+  margin: 27px auto 0 auto;
+  width: 140px;
+  height: 48px;
+  padding: 12px 16px;
+  font-weight: 500;
+  font-size: 17px;
+  background-color: #fff;
+  color: #6070ff;
+  border-radius: 8px;
+  display: flex;
+}
+
+.get-it-touch:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.get-it-touch:visited {
+  background: #2230d2;
+  color: #fff;
+}
+
+@media (min-width: 992px) {
+  header article,
+  main > section {
+    width: calc(100% - 60px);
+    margin: 0 auto;
+  }
+}
```

Comment:
- [ ] Please note that on screen size 1024px the design is missing the required margin on the work section and the decorations on the headline section are also not seen👀, Kindly fix this by using a responsive CSS styling✌
- Your design
![screenshot (55)](https://user-images.githubusercontent.com/44624138/144619051-f2e53333-0977-4f6b-b197-10c0e518e757.png) 
- Figma design
![screenshot (54)](https://user-images.githubusercontent.com/44624138/144619548-c9d60c83-7fac-4944-8fc1-c3f98bcd1514.png)

Reply:
- [ ] Kindly fix the broken image kink as well for better `User-Interface`😉

Reply:
> 

thanks for your feedback. actually the images are working on mine I couldn't know what happened there, is putting a screenshot about the project is enough instead of live demo?